product-page/styles/product.css

```diff
@@ -1,10 +1,43 @@
-:root {
-  --color-light-grey: #e2e8f0;
+@import "/logout/logout.css";
+
+.sidebar {
+  color: var(--color-light-grey);
+  padding: 1.6rem;
+}
+
+.documentation {
+  grid-template-columns: 25rem auto;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1rem 0;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+  border: 0;
+  margin: 8px 8px;
+}
+
+input[type="range"] {
+  width: 16rem;
+  margin: 8px 8px;
```

Comment:
changes: you can use 8px once here.

Reply:
Since it's same thing, I think it's fine to keep it this way. Thanks for your feedback.